### DISCUSSION TITLE
Is it possible to place Slider component vertically

### DISCUSSION THREAD

**Discussion by subhasishk23** [ArthurSonzogni/FTXUI#466] [Q&A] [answered]
Is it possible to place Slider component vertically?
The example code provided at examples/component/focus.cpp renders Slider horizontally.
I want to place it vertically to look like a scroller.

**Comment by subhasishk23** [comment]
I am new to this library. Not able to figure out scroll option for elements.

**Comment by ArthurSonzogni** [comment, marked as answer]
It would be very nice to add slider for every directions. I will take a look.

**Reply by subhasishk23** [reply]
Thanks a lot for considering this.

**Reply by ArthurSonzogni** [reply]
This has been fixed!
